README.rst

```diff
@@ -47,6 +47,9 @@ Here's an example configuration file:
        "redis_url": "redis://127.0.0.1/5",
        "expire_keys": 900
      },
+     "store_cache": {
```

Comment:
I don't think the `StoreCache` abstraction should leak out into the configuration. Instead, we should have the different `expire_` config values under `store`, similar to what you had before.